docs/user-guide/release-notes/1.2.x.rst

```diff
@@ -32,3 +32,29 @@ Bugs Fixed
 ----------
 
 You can see the :fixedbugs:`list of bugs fixed<1.2.0>`.
+
+
+Pulp OSTree 1.2.1
+=================
+
+New Features
+------------
+
+- Support has been added to `repair` a corrupted :term:`local` repository.
+  Using the optional `--repair` or `-r` option the importer will repair the corrupted repository.
+  The repair may require that the repository be deleted and recreated. This option
```

Comment:
I think it's worth using a different word than "repository" here, to make it clear that the Pulp repo will not be changed. Maybe "content on disk" or something like that?